### DISCUSSION TITLE
Team 3 Lab 1

### DISCUSSION THREAD

**Discussion by emmats** [atsa-es/fish550-2023#10] [General]
I'm going to start by making the last 5 years of the data the test set, so years 1952-2010 will be the training set.
